polus-imagej-filter-dog-plugin/plugin.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "ImageJ filter dog",
+  "version": "0.2.11",
+  "title": "ImageJ filter dog",
+  "description": "Difference of Gaussians (DoG) implementation",
+  "author": "Benjamin Houghton (benjamin.houghton@axleinfo.com)",
+  "institution": "National Center for Advancing Translational Sciences, National Institutes of Health",
+  "repository": "https://github.com/labshare/polus-plugins",
+  "website": "https://ncats.nih.gov/preclinical/core/informatics",
+  "citation": "",
+  "containerId": "polusai/imagej-filter-dog-plugin:0.2.11",
+  "inputs": [
+    {
+      "name": "opName",
+      "type": "enum",
+      "options": {
+        "values": ["DoGSingleSigmas",
+                    "DoGVaryingSigmas"]
+      },
+      "description": "Imagej op overloading method to perform",
+      "required": true
+    },
+    {
+      "name": "inpDir",
+      "type": "collection",
+      "description": "Input image collection to be processed by this plugin",
+      "required": true
+    },
+    {
+      "name": "sigma1",
+      "type": "number",
+      "description": "A single value for sigma1",
+      "required": false
+    },
+    {
+      "name": "sigma2",
+      "type": "number",
+      "description": "A sigle value for sigma2",
+      "required": false
+    },
+    {
+      "name": "sigmas1",
+      "type": "array",
+      "description": "sigmas1",
```

Comment:
Same comment as above.

Reply:
Description has been added for this input